app/services/metrics/metrics.ts

```diff
@@ -0,0 +1,37 @@
+import { Service } from 'services/core/service';
+import { ipcRenderer } from 'electron';
+import { Inject } from '../core';
+import { PerformanceService } from 'services/performance';
+
+export interface IMetricsState {
+  appStartTime: number;
+  mainWindowShowTime: number;
+  sceneCollectionLoadingTime: number;
+}
+
+/**
+ * An utility service that keeps the time of some key events to help find performance issues
+ * This services is used in performance tests
+ */
+export class MetricsService extends Service {
+  @Inject() performanceService: PerformanceService;
+
+  protected init() {
+    const appStarTime = ipcRenderer.sendSync('getAppStartTime');
```

Comment:
Should we be disabling this service in production?  Don't want to be a doing a synchronous IPC on startup in production for no reason.